### DISCUSSION TITLE
Contacts->Person needs free text Salutation field

### DISCUSSION THREAD

**Discussion by nick-prater** [ledgersmb/LedgerSMB#5869] [Ideas]
While the current dropdown, populated from values in the salutation table is convenient and aims to minimise data entry errors, we cannot hope to list all salutations and their possible combinations, along with international versions.

Consider a small selection of those we currently miss:
Lord, Lady, Reverend, Colonel, Herr, Frau, Señor, Rt Hon, Rt Hon Lord, Rt Hon Dr....

The salutation field should therefore allow free text entry.

**Comment by sbts** [comment]
I sort of agree, however that way leads to anarchy with multiple variations of the same salutation getting entered into the system over time.

Instead, I'd suggest a dropdown list backed by a db table.
Any "manually entered" Salutation is automatically added to the table so it can be selected next time rather than entered.
